locales/fr.lua

```diff
@@ -0,0 +1,20 @@
+local Translations = {
+    error = {
+        invalid_ext = "Ce n'est pas une extension valide, seul",
```

Comment:
I don't think you finished this sentence?

Reply:
It's not finished, 
```suggestion
        invalid_ext = "Ce n'est pas une extension valide, seul les liens d'extension %{fileext} sont autorisés",
```

Reply:
> 

yup forgor to update this one